### PR TITLE
🧹Admin Panel Revamped

### PR DESCRIPTION
**Pull request by Yug-gurnani**

### CODE REVIEW

**Review by kaydee0502 — APPROVED** [review]

**Review by LakshitXD — APPROVED** [review]

**Review by Adhikram — APPROVED** [review]

File: devsnest/app/resources/api/v1/admin/internal_feedback_resource.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      # Resource
+      class InternalFeedbackResource < JSONAPI::Resource
+        attributes :id, :user_id, :user_name, :is_resolved, :issue_type, :issue_described, :feedback, :issue_scale
+
+        filter :span, apply: lambda { |records, value, _options|
```
Comment (vedansh) [inline devsnest/app/resources/api/v1/admin/internal_feedback_resource.rb:10]: span is being send from the frontend?

File: devsnest/app/controllers/api/v1/group_members_controller.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@ def update_user_group
           # User has been removed from a team
           current_group_member.destroy
         end
-
+        user.update(group_assigned: true)
```
Comment (vedansh) [inline devsnest/app/controllers/api/v1/group_members_controller.rb:57]: can you create a small script to update all existing users who are part of the group

File: devsnest/app/resources/api/v1/admin/internal_feedback_resource.rb (hunk 1 above)
Comment (Yug-gurnani) [reply devsnest/app/resources/api/v1/admin/internal_feedback_resource.rb:10]: Yes, in the params

File: devsnest/spec/requests/admin/internal_feedback_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      expect(response.status).to eq(401)
+    end
+  end
+
+  context 'Internal Feedback - filter check' do
+    it 'Filter should be applied' do
+      user.update(user_type: 1)
+      get "/api/v1/admin/internal-feedback?filter[span]=#{Date.today}"
+      expect(response.status).to eq(200)
+    end
```
Comment (vedansh) [inline devsnest/spec/requests/admin/internal_feedback_spec.rb:29]: data bhi check kar lo, ki response mai data aa raha hai ki nahi and sahi data aa raha hai ki nahi

**Review by LakshitXD — APPROVED** [review]

**Pull request merged**